src/v2/Apps/ViewingRoom/ViewingRoomsApp.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
```

Comment:
Oops.. Forgot that I made the branch on top of other work. But the other work is mergeable as well.